<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>U2::SaveSelectedSequenceFromMSADialog</class>
 <widget class="QDialog" name="U2::SaveSelectedSequenceFromMSADialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>195</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export selected sequence from alignment</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="fileLabel">
       <property name="text">
        <string>Export to file</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="fileNameEdit"/>
     </item>
     <item row="0" column="2">
      <widget class="QToolButton" name="fileButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="formatLabel">
       <property name="text">
        <string>File format to use</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QComboBox" name="formatCombo"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="addToProjectBox">
     <property name="text">
      <string>Add document to the project</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gapGroup">
     <property name="title">
      <string>Gap characters ('-')</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QRadioButton" name="keepGapsRB">
        <property name="text">
         <string>Keep</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="trimGapsRB">
        <property name="text">
         <string>Trim</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>U2::SaveSelectedSequenceFromMSADialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>U2::SaveSelectedSequenceFromMSADialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
